specification.md

```diff
@@ -159,6 +159,7 @@ message AgentToServer {
     AgentAddonStatuses addon_statuses = 3;
     AgentInstallStatus agent_install_status = 4;
     AgentDisconnect agent_disconnect = 5;
+    AgentToServerFlags flags = 6;
```

Comment:
ServerToAgent has "Flags", so I used "AgentToServerFlags" here but maybe both of them could use a better names. What do you think @tigrannajaryan ?

Reply:
AgentToServerFlags sounds good to me. If you can think of a better name please suggest.